### ISSUE TITLE
Compare TensorFlow backend performance to Theano

### ISSUE THREAD

**Issue by tschimbr** [eonum/medcodelearn#18]
Speed and accuracy

**Comment by tschimbr** [comment]
Both on CPU and GPU

**Comment by tschimbr** [comment]
Theano with CPU on HP laptop: PDX task only. 10 epochs. lstm embedding pretrained: 42m / 26.12 %

**Comment by tschimbr** [comment]
conda install -c https://conda.anaconda.org/jjhelmus tensorflow

**Comment by tschimbr** [comment]
TensorFlow with CPU on HP laptop: PDX task only. 10 epochs. lstm embedding pretrained: 30m / 26.41%

**Comment by tschimbr** [comment]
Theano on GPU (GeForce GTX 560 (CNMeM is disabled, CuDNN not available)). PDX task only. 10 epochs. lstm pretrained embedding: 7m29s / 27.63%

**Comment by tschimbr** [comment]
TensorFlow on GPU (GeForce GTX 560 (CNMeM is disabled, CuDNN not available)). PDX task only. 10 epochs. lstm pretrained embedding: 15m42s / 28.16%

**Comment by tschimbr** [comment]
TensorFlow not yet working with GPU GeForce GTX 560!
